src/shared/components/OrderList.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="order-list">
     <ion-grid>
-      <ion-row v-for="order in purchaseOrderSequences" :key="order.id">
+      <ion-row v-for="order in purchaseOrderSequences" :key="order.id" @load="timeFormater">
         <ion-col> {{ order.id }} </ion-col>
         <ion-col> {{ order.description }} </ion-col>
         <ion-col> {{ order.date }} </ion-col>
```

Comment:
```suggestion
        <ion-col> {{ order.date | date: 'dd-mm-yyyy' }} </ion-col>
```
Use filters for vue https://vuejs.org/v2/guide/filters.html

Reply:
done